src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -680,6 +733,31 @@ pub fn new_sub_level(
     }
 }
 
+pub fn add_ssts_to_sub_level(
+    l0: &mut OverlappingLevel,
+    sub_level_idx: usize,
+    insert_sub_level_id: u64,
+    level_type: LevelType,
+    insert_table_infos: Vec<SstableInfo>,
+) {
+    if sub_level_idx < l0.sub_levels.len() {
+        l0.sub_levels[sub_level_idx]
+            .table_infos
+            .extend(insert_table_infos);
+        if l0.sub_levels[sub_level_idx].level_type == LevelType::Nonoverlapping as i32 {
+            l0.sub_levels[sub_level_idx]
+                .table_infos
+                .sort_by(|sst1, sst2| {
+                    let a = sst1.key_range.as_ref().unwrap();
+                    let b = sst2.key_range.as_ref().unwrap();
+                    a.compare(b)
+                });
+        }
+        return;
```

Comment:
sub level's stats (e.g. total size) is not set.

Reply:
fixed